pgo/cmd/test.go

```diff
@@ -108,16 +110,28 @@ func showTest(args []string, ns string) {
 
 		for _, result := range response.Results {
 			fmt.Println("")
-			fmt.Printf("cluster : %s \n", result.ClusterName)
-			for _, v := range result.Items {
-				fmt.Printf("%s%s is ", TreeBranch, v.PsqlString)
-				if v.Working {
-					fmt.Printf("%s\n", GREEN("Working"))
-				} else {
-					fmt.Printf("%s\n", RED("NOT working"))
-				}
-			}
+			fmt.Println(fmt.Sprintf("cluster : %s", result.ClusterName))
+
+			// first, print the test results for the endpoints
+			printTestResults("Endpoints", result.Endpoints)
+			// first, print the test results for the instances
```

Comment:
I suggest using `Services` for the header here instead of `Endpoints`, e.g.:

```bash
cluster : mycluster2
        Services
                primary (10.104.67.174:5432): UP
```

This is because validating the addresses in the `endpoint` is really validating that the selector defined in the service actually matches a pod, i.e. that the endpoint contains what should be the IPs for the primary, replica, pgbouncer, etc. pods, effectively making them available via the service.  So really this is validating that the service has been properly configured to send traffic to the appropriate pod (which also results in the automatic creation of the proper endpoints).

Also the IP address displayed in the output is the `ClusterIP` of the service and not an endpoint (i.e. Pod) IP, and since the `endpoint` is typically invisible to the end user and the `service` is the resource a user would interact with directly, it seems appropriate to show that the `service` is up vs. the `endpoint`.